spec/controllers/kalibro_configurations_controller_spec.rb

```diff
@@ -1,6 +1,14 @@
 require 'rails_helper'
 
 describe KalibroConfigurationsController, :type => :controller do
+  
+  def post_method(method)
+    unless method == :create
+      post method, :id => kalibro_configuration.id, :kalibro_configuration => kalibro_configuration_params, :attributes => attributes
+    else
+      post method, :kalibro_configuration => kalibro_configuration_params, :attributes => attributes
```

Comment:
I don't think this method is necessary.
It is only used for the create and update actions. But the correct method for update is PUT, right?